### PR TITLE
feat: 初始化项目内容

### PR DESCRIPTION
**Pull request by juzhiyuan**
初始化项目结构，语言无关。

在此访问分支目录 https://github.com/api7/template-repo/tree/feat-initial

### CODE REVIEW

**Comment by juzhiyuan** [comment]
image

has been uninstalled due to the paid plan.

**Comment by juzhiyuan** [comment]
Let's retrigger this PR's checks.

**Comment by juzhiyuan** [comment]
emm just ignored that pending check.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--
```
Comment (spacewander) [inline CHANGELOG.md:1]: We don't need a CHANGELOG as the template? Not all repo (like libraries) need a change log.

Comment (juzhiyuan) [reply CHANGELOG.md:1]: Not sure 🤔  If we create a dependency repo, we may need this file.

**Review by membphis — APPROVED** [review]
LGTM ^_^

**Comment by membphis** [comment]
@juzhiyuan please use English title

**Comment by juzhiyuan** [comment]
> @juzhiyuan please use English title

API7 也要吗？

**Comment by membphis** [comment]
> > @juzhiyuan please use English title
> 
> API7 也要吗？

we'd better follow this way. Our goal is to become an international company, and there will be foreign partners joining.

image

File: CHANGELOG.md (hunk 1 above)
Comment (juzhiyuan) [reply CHANGELOG.md:1]: Just removed them if they are not needed after the repo is created

**Review by membphis — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Ping @moonming

**Comment by moonming** [comment]
The license is good for ASF projects, but if for api7, we need to change the asf header.
Others LGTM

**Comment by juzhiyuan** [comment]
> The license is good for ASF projects, but if for api7, we need to change the asf header.
> Others LGTM

How about renaming this template repo to template-apache-repo?

**Comment by juzhiyuan** [comment]
Because in Apache repo, we need some actions and checkers to make sure every file's license is valid.

**Comment by moonming** [comment]
ok

**Comment by juzhiyuan** [comment]
Ok let me update some text tomorrow

**Comment by juzhiyuan** [comment]
Just updated this repo's name to `template-apache-repo`

**Comment by juzhiyuan** [comment]
ping @moonming

**Review by moonming — APPROVED** [review]

**Pull request merged**